source/renderer/app/components/news/AlertsOverlay.js

```diff
@@ -52,12 +52,12 @@ export default class AlertsOverlay extends Component<Props, State> {
   onClose = () => {
     const { alerts } = this.props;
```

Comment:
Not important, but why not `const { onMarkNewsAsRead, onCloseOpenAlert, alerts } = this.props` and use short names for subsequent function calls?

Reply:
@yakovkaravelov aha I can do that. This part came from the past and I kept that as is.